paper/writeup.tex

```diff
@@ -1256,6 +1256,9 @@ \section{Discussion}
 On the other hand, locally building trees separately in each segment of genome as is done by Relate \cite{speidel2019method}
 simply does not provide this sort of haplotype-based information (although it might be added using a method like ours).
 
+\section{Acknowledgments}
+The authors would like to thank Dr. Yan Wong for his suggestions and insights for ARGs with locally-unary nodes.
```

Comment:
Maybe ".. for helpful comments anad suggestions" here?

Reply:
I can do that